src/sys/event.rs

```diff
@@ -51,7 +54,15 @@ libc_enum! {
         #[cfg(any(target_os = "ios", target_os = "macos"))]
         EVFILT_MACHPORT,
         EVFILT_PROC,
+        /// Returns events associated with the process referenced by a given
+        /// process descriptor, created by pdfork(2). The events to monitor are:
+        /// 
+        /// - NOTE_EXIT: the process has exited. The exit status will be stored in data.
+        #[cfg(target_os = "freebsd")]
+        EVFILT_PROCDESC,
         EVFILT_READ,
+        #[cfg(target_os = "freebsd")]
+        EVFILT_SENDFILE,
```

Comment:
@asomers Could you help us dig up some docs for this constant?

Reply:
How about "indicates the completion of an asynchronous `sendfile(2)` call".  I'm afraid be more specific than that.